api/order.go

```diff
@@ -162,17 +186,45 @@ func (a *API) setUserIDFromToken(tx *gorm.DB, user *models.User, order *models.O
 	return nil
 }
 
-func (a *API) createLineItems(tx *gorm.DB, order *models.Order, items []models.LineItem) *HTTPError {
+func (a *API) createLineItems(ctx context.Context, tx *gorm.DB, order *models.Order, items []*OrderLineItem) *HTTPError {
+	sem := make(chan int, MaxConcurrentLookups)
```

Comment:
Probably a good idea yeah, as long as there's a sensible default (shouldn't really be something normal people should have to worry about.)